TypeScript issue in @react-spring/shared for `Rafz`

🙏  Thank you for the latest release of 9. Love it ❤️  .
---

I am experiencing an issue when using it in TypeScript. Following are the details of the issue and possible fixes.

### Description

|Package|Version|
|:-|-:|
|@react-spring/web|9.2.2|
|typescript|4.0.2|

I am using `@react-spring/web` in one of my packages. When exporting type definitions, I got following error:

`Cannot find module 'packages/rafz' or its corresponding type declarations.`

Here is how I am generating type declaration for my package.

```sh
$ tsc src/*.tsx --outDir types --emitDeclarationOnly --declaration true --jsx preserve --target es5 --allowSyntheticDefaultImports true
.
../../node_modules/@react-spring/shared/dist/declarations/src/globals.d.ts(3,22): error TS2307: Cannot find module 'packages/rafz'
or its corresponding type declarations.
.
```

After inspection, I found that type definitions of `Rafz` were imported from `packages/rafz` instead of `@react-spring/rafz`

https://github.com/pmndrs/react-spring/blob/81ba96ce89fcec1391136a66e2f0de25ae0eb1c3/packages/shared/src/globals.ts#L1-L7

### Possible Solution

This is a small import issue which can be easily fixed by importing type definitions of `Rafz` from `@react-spring/rafz`.

```diff
- import { raf } from '@react-spring/rafz' 
+ import { raf, Rafz } from '@react-spring/rafz' 
.
.
- import { Rafz } from 'packages/rafz' 
```

 I tested it locally by updating the declaration files in my `node_modules` and it is working

----

I would love to contribute this fix if this issue seems like a bug, or 🙏  please guide for possible fixes if this is not a but.

--- a/packages/shared/src/globals.ts
+++ b/packages/shared/src/globals.ts
@@ -1,10 +1,9 @@
-import { raf } from '@react-spring/rafz'
+import { raf, Rafz } from '@react-spring/rafz'
 import {
   OneOrMore,
   InterpolatorConfig,
   InterpolatorArgs,
 } from '@react-spring/types'
-import { Rafz } from 'packages/rafz'
 
 import { FluidValue } from './fluids'
 import type { OpaqueAnimation } from './FrameLoop'

